### PR TITLE
Create database entities

### PR DESCRIPTION
**Pull request by tmorris42**
Create starting database entities for:
- User
- Message
- Channel
- Karma

Re: #2

### CODE REVIEW

File: backend/src/users/user.entity.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ unique: true })
+  identity: string;
+
+  @Column({ unique: true })
+  username: string;
+
+  @Column({ type: 'bytea' })
```
Comment (fyusuf-a) [inline backend/src/users/user.entity.ts:21]: Is that a typo?

File: backend/src/users/user.entity.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    joinColumn: {
+      name: 'blocker',
+      referencedColumnName: 'id',
+    },
+    inverseJoinColumn: {
+      name: 'blockee',
+      referencedColumnName: 'id',
+    },
+  })
+  blocked: User;
```
Comment (fyusuf-a) [inline backend/src/users/user.entity.ts:61]: Shouldn't it be `blocked: User[]` ?

File: backend/src/messages/message.entity.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import { User } from 'src/users/user.entity';
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class Message {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  contest: string;
```
Comment (fyusuf-a) [inline backend/src/messages/message.entity.ts:11]:
```suggestion
  content: string;
```

File: backend/src/users/user.entity.ts (hunk 2 above)
Comment (tmorris42) [reply backend/src/users/user.entity.ts:61]: Yes!
```suggestion
  blocked: User[];
```

**Review by fyusuf-a — APPROVED** [review]
Thank you.

**Pull request merged**